includes/admin/payments/class-refund-items-table.php

```diff
@@ -140,11 +136,71 @@ protected function get_primary_column_name() {
 	public function column_default( $order_item, $column_name ) {
 		switch ( $column_name ) {
 			case 'amount':
-			case 'discount':
-			case 'tax':
-			case 'subtotal':
-			case 'total':
 				return $this->format_currency( $order_item, $column_name );
+
+			case 'total' :
+				return $this->format_currency( $order_item, $column_name, 0 );
+
+			case 'quantity' :
+				ob_start();
+				?>
+				<label for="edd-order-item-quantity-<?php echo esc_attr( $order_item->id ); ?>" class="screen-reader-text">
+					<?php esc_html_e( 'Quantity to refund', 'easy-digital-downloads' ); ?>
+				</label>
+				<input type="number" id="edd-order-item-quantity-<?php echo esc_attr( $order_item->id ); ?>" class="edd-order-item-refund-quantity edd-order-item-refund-input" name="refund_order_item[<?php echo esc_attr( $order_item->id ); ?>][quantity]" value="0" min="0" max="<?php echo esc_attr( $order_item->quantity ); ?>" step="1""<?php echo 'refunded' === $order_item->status ? ' disabled' : ''; ?> />
+				<?php
+				return ob_get_clean();
+
+			case 'subtotal' :
+			case 'tax' :
+				$currency_pos = edd_get_option( 'currency_position', 'before' );
+
+				// Maximum amounts that can be refunded.
+				$refundable_amounts = $order_item->get_refundable_amounts();
+				$amount_remaining   = array_key_exists( $column_name, $refundable_amounts ) ? $refundable_amounts[ $column_name ] : $order_item->{$column_name};
+
+				/*
+				 * Original amount.
+				 * For subtotals, we actually do subtotal minus discounts for simplicity so that the end user
+				 * doesn't have to juggle that.
+				 */
+				$original_amount = $order_item->{$column_name};
+				if ( 'subtotal' === $column_name ) {
+					$original_amount -= $order_item->discount;
+				}
+				ob_start();
+				?>
+				<label for="edd-order-item-<?php echo esc_attr( $order_item->id ); ?>-refund-<?php echo esc_attr( $column_name ); ?>" class="screen-reader-text">
+					<?php
+					if ( 'subtotal' === $column_name ) {
+						_e( 'Amount to refund, excluding tax', 'easy-digital-downloads' );
+					} else {
+						_e( 'Amount of tax to refund', 'easy-digital-downloads' );
+					}
+					?>
+				</label>
+				<?php
+				if ( 'before' === $currency_pos ) {
+					echo esc_html( $this->get_currency_symbol( $order_item->order_id ) );
+				}
+				?>
+				<input type="text" id="edd-order-item-<?php echo esc_attr( $order_item->id ); ?>-refund-<?php echo esc_attr( $column_name ); ?>" class="edd-order-item-refund-<?php echo esc_attr( $column_name ); ?> edd-order-item-refund-input" name="refund_order_item[<?php echo esc_attr( $order_item->id ); ?>][<?php echo esc_attr( $column_name ); ?>]" value="<?php echo esc_attr( edd_sanitize_amount( 0 ) ); ?>" data-original="<?php echo esc_attr( $original_amount ); ?>" data-max="<?php echo esc_attr( $amount_remaining ); ?>" <?php echo 'refunded' === $order_item->status ? ' disabled' : ''; ?> />
```

Comment:
Should this be `type="number"` instead? If it needs to be `text`, which may be the case for differing currency separators (?), I think it needs to have `small-text` added as a class, although that's potentially too small. Noticing due to issues with responsiveness--we may actually just need to apply custom CSS for this to work with the currency symbol in the table.